newsroom/templates/scripts.html

```diff
@@ -35,35 +35,36 @@
 <!-- Global site tag (gtag.js) - Google Analytics -->
 <script async src="https://www.googletagmanager.com/gtag/js?id={{ get_setting('google_analytics') | safe }}"></script>
 <script>
-  window.dataLayer = window.dataLayer || [];
-  function gtag(){dataLayer.push(arguments);}
-  gtag('js', new Date());
+    window.dataLayer = window.dataLayer || [];
+    function gtag(){dataLayer.push(arguments);}
+    gtag('js', new Date());
 
-  gtag('set', 'user_properties', {
-      company: (window.profileData || {}).companyName || 'none',
-      user: ((window.profileData || {}).user || {}).first_name || 'unknown',
-  });
+    gtag('set', 'user_properties', {
+        company: (window.profileData || {}).companyName || 'none',
+        user: ((window.profileData || {}).user || {}).first_name || 'unknown',
+    });
 
-  {% if session.get('user') %}
-  gtag('set', {'user_id': '{{ hash(session['user']) | tojson }}' });
-  {% endif %}
+    let configParams = {};
+    {% if session.get('user') %}
+        gtag('set', {'user_id': '{{ hash(session['user']) | tojson }}' });
 
-  if ('{{ get_setting('google_analytics') }}'.startsWith('UA')) {
-      gtag('config', '{{ get_setting('google_analytics') | tojson }}', {
-          custom_map: {
-              dimension1: 'company',
-              dimension2: 'user',
-          }
-      });
-  } else {
-      gtag('config', '{{ get_setting('google_analytics') | tojson }}');
-  }
-
-  {% with messages = get_flashed_messages(category_filter=['analytics']) %}
-    {% if messages %}
-        analytics.sendEvents({{ messages | tojson | safe }});
+        configParams['user_id'] = '{{ session['user'] }}';
     {% endif %}
-  {% endwith %}
+
+    if ('{{ get_setting('google_analytics') }}'.startsWith('UA')) {
```

Comment:
you can drop this UA section, that won't be supported by Google after July 1st

Reply:
done